src/wizard.scss

```diff
@@ -0,0 +1,300 @@
+@import "./mixins";
+
+// VARIABLES
+$block: #{$fd-namespace}-wizard;
+
+$fd-wizard-step-indicator-spacing: 0.1875rem !default;
+$fd-wizard-step-indicator-size: 2rem !default;
+$fd-wizard-step-no-label-max-size: 4rem !default;
+$fd-wizard-step-indicator-stacked-offset: -1 * ($fd-wizard-step-indicator-size - $fd-wizard-step-indicator-spacing + 0.25rem) !default;
+$fd-wizard-stacked-upcoming-step-offset: 2.375rem !default;
+
+@mixin fd-wizard-responsive-paddings() {
+  &--s {
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+
+  &--m,
+  &--l {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  &--xl {
+    padding-left: 3rem;
+    padding-right: 3rem;
+  }
+}
+
+// BLOCK
+.#{$block} {
+  @include fd-reset();
+
+  position: relative;
+
+  &__navigation {
+    @include fd-reset();
+  }
+
+  &__progress-bar {
+    @include fd-reset();
+    @include fd-wizard-responsive-paddings();
```

Comment:
What do you think about moving `--s/m/l/xl` to block and applying padding on progress bar same as for content

Reply:
Could you be more specific? I don't understand your idea